.github/workflows/env-setup-init-redis-cluster-redis.yml

```diff
@@ -0,0 +1,37 @@
+name: Env Setup - Initialize Redis Cluster in Openshift
+run-name: Env Setup - Initialize Redis Cluster in Openshift from ${{ inputs.gitRef }} to ${{ inputs.environment }}
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      gitRef:
+        description: "Build Ref"
+        required: true
+        default: ""
+      redisPort:
+        description: "Redis Port"
+        required: false
+        default: 6379
```

Comment:
i believe the port 6379 is used to create a cluster and not create redis with the port 6379.

Reply:
That is to initialize the cluster.

Reply:
I removed that input. That's not required and the make command will use 6379 by default.

Reply:
IMO input for port is not a concern as of now unless we have requirement to run redis on a different port other than 6379.